doc/examples/jupyter_notebook/Makefile

```diff
@@ -1,12 +1,11 @@
 build:
 	docker build -t pachyderm_jupyter .
 
-prep: build
+prep:
 	pachctl version
-	./citibike_data/load.sh
-	./weather_data/load.sh
+	cd citibike_data && ./load.sh
+	cd weather_data && ./load.sh
 	pachctl create-pipeline -f ./sales_data/pipeline.json
-	pachctl create-job -f jupyter.json
```

Comment:
As we discussed in Slack, this is slightly more complicated because the user needs to flush commit and place the right commit id into the job spec.  This is documented in the readme, so I think shouldn't automate this for now.  Although, I guess we could use `sed` to replace it in the spec or something.